src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/ProjectDependenciesSubTreeProvider.cs

```diff
@@ -90,6 +90,22 @@ protected override IDependencyNode CreateRootNode()
                                              resolved: resolved);
         }
 
+        public override IDependencyNode GetDependencyNode(DependencyNodeId nodeId)
+        {
+            if (nodeId == null)
+            {
+                return null;
+            }
+
+            var projectNode = RootNode.Children.FirstOrDefault(x => x.Id.ToNormalizedId().Equals(nodeId));
+            if (projectNode != null)
+            {
+                projectNode.AddChild(projectNode);
+            }
+
+            return projectNode;
+        }
```

Comment:
This is not needed for this change, and was part of another change i am working on. Will revert it when add unit tests in next commit